open-banking-brasil-financial-api-1_ID1.md

```diff
@@ -251,8 +252,8 @@ This profile defines "cnpj" as a new standard claim as per
 
 **CNPJ**, short for Cadastro Nacional de Pessoas Jurídicas, is an identification number
  of **Brazilian** companies issued by the **Brazilian** Ministry of Revenue, **in**
- Portuguese "Secretaria da Receita Federal" or "Ministério da Fazenda". In the Brasil open banking identity model,
- individuals can associated with 0 or more CNPJs. A CNPJ is an number that is either 8 or 14 digits long,
+ Portuguese "Secretaria da Receita Federal" or "Ministério da Fazenda". In the Brasil Open Banking identity model,
+ individuals can associated with 0 or more CNPJs. A CNPJ is a string consisting of numbers that is either 8 or 14 digits long and may start with a 0,
```

Comment:
Change text to:

**(change starts)**
individuals can be associated with 0 or more CNPJs. A CNPJ is a string consisting of numbers 14 digits long and may start with one or more 0,
**(change ends)**

Just to clarify the 8 digits code is called ISPB code which is a code used by the central bank to define an organization.

It is composed by the 8 first digits of the CNPJ, however it's not exactly called CNPJ